### DISCUSSION TITLE
Drop downs were superior to -/+ for Trains

### DISCUSSION THREAD

**Discussion by hypercube33** [OpenRCT2/OpenRCT2#20609] [Feature requests] [answered]
With the new - + buttons you cant tell what the combination of cars to train count is without clicking a ton of times. The old way was the best.

**Comment by TheClaudeQC** [comment, marked as answer]
maybe only whenever you don't have the cheat that allow you to have as many trains or cars per train as you which?

**Reply by Gymnasiast** [reply]
That, indeed. Nobody wants a dropdown with 256 items.

Also, hovering over the steppers shows a tooltip with the maximum number of trains and cars per train, so it’s not like one is missing information.
